### ISSUE TITLE
Send chat macro doesn't work now

### ISSUE THREAD

**Issue by MarTyNiDruid** [rivafarabi/deckboard#263]
Hey,

the send chat macro doesn't work now for some reason. You can click it but it just doesn't do anything. Nothing shows up in chat. (It was working for me previously)

**Comment by rivafarabi** [comment]
can you send me the log.log file located at C:\Users{USER}\AppData\Roaming\Deckboard?

**Comment by MarTyNiDruid** [comment]
Yeah, sorry for the hoax. It started working after a pc restart. Not sure what happened :/ 

But let me take this opportunity to thank you for this amazing app! I love it and use it every stream for various things! I'm only waiting for the stream markers to start working (they're not working for me yet at least) and then it's gonna be a perfect streamdeck alternative app! Thanks for your hard work!
